validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -79,25 +116,51 @@ def __init__(self, block_cache, state_view, data_dir):
         self._state_view = state_view
 
     def verify_block(self, block_state):
-        return True
+        if block_state.header.consensus == b"Devmode":
+            return True
+        else:
+            return block_state.header.consensus == b"Devmode"
 
```

Comment:
These four lines can be condensed down to simply:

return block_state.header.consensus == b"Devmode"